spec/marten/cli/manage/command/new_spec.cr

```diff
@@ -173,6 +173,21 @@ describe Marten::CLI::Manage::Command::New do
       end
     end
 
+    it "creates a new project structure from edge" do
+      stdout = IO::Memory.new
+
+      command = Marten::CLI::Manage::Command::New.new(
+        options: ["--edge", "project", "dummy_project"],
+        stdout: stdout
+      )
+
+      command.handle
+
+      Marten::CLI::Manage::Command::NewSpec::PROJECT_FILES_WITH_EDGE.each do |path|
```

Comment:
I think we can probably avoid defining a new `PROJECT_FILES_WITH_EDGE` constant here (in order to avoid duplications!). Instead, we can probably use the existing `PROJECT_FILES` constant to which we append the additional `shard.override.yml`.

For example:

```suggestion
      (
        Marten::CLI::Manage::Command::NewSpec::PROJECT_FILES +
        ['shard.override.yml']
      ).each do |path|
```